odfuzz/fuzzer.py

```diff
@@ -8,6 +8,7 @@
 import gevent
 import requests
 import requests.adapters
+import json
```

Comment:
I would prefer to not use json module in already "too much of everything" fuzzer.py. Its classes should IMHO grab something from somewhere (which correctness is unit tested in isolation) and just tie them together. 

Since in effect it is only about one line at  608:body = json.dumps(body), it would be easy to move away.

Also relevant to the comment export in the docs about JSON/XML-Atom